src/lib/optimizer/strategy/chunk_pruning_rule.cpp

```diff
@@ -87,21 +88,47 @@ std::set<ChunkID> ChunkPruningRule::_compute_exclude_list(
   std::set<ChunkID> result;
 
   for (const auto& operator_predicate : *operator_predicates) {
+    // Cannot prune column-to-column predicates atm. Column-to-placeholder predicates are never prunable.
```

Comment:
atm?

Reply:
"Cannot prune column-to-column predicates" would mean there is some fundamental impossibility in pruning chunks with column-vs-column predicates - which is not the case (though it might be complicated) 

Add the "atm" and it becomes clear that the current code base of hyrise just doesn't do it.

How would you formulate it?

Reply:
or did you take issue with the abbreviation? i am all for compact commenting but not if it leaves me guessing too much ;)

Reply:
> or did you take issue with the abbreviation?

Yepp, sorry.